components/pre_trial_pricing_modal.tsx/pre_trial_pricing_modal.scss

```diff
@@ -0,0 +1,277 @@
+.PreTrialPricingModal {
+    .modal-dialog {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        margin: auto;
+        transform: translate(-50%, -50%) !important;
+        width: 1440px;
+
+        .modal-content {
+            background-color: var(--center-channel-bg);
+        }
+    }
+
+    .modal-header,
+    .modal-footer {
+        padding: 24px;
+    }
+
+    .modal-body {
+        overflow: visible;
+        max-height: none;
+        padding: 0;
+    }
+
+    .self-hosted-alert {
+        position: absolute;
+        right: 160px;
+        top: 37px;
+        height: 40px;
+
+        h1 {
+            color: #3F4350;
```

Comment:
There's a number of hex codes in here. Some have matching variables names. Should they use those variable names?

`#DADFE5` has no prior use in the codebase, and only one use in customer-web-server (with no variable name). It does match what's in the figma (border bottom right below the "Free"/"$10/user/month"/"Contact Sales"). @annelieseklein is this a new color in the palette we should add with a specific name? Or should we just leave it as a one-off?

Reply:
Put the repeated ones in variables